### PR TITLE
I386

### PR DESCRIPTION
**Pull request by nickdesaulniers**
i386 support
    
cc #17
    
Still need to add check_logs.py support once
https://github.com/ClangBuiltLinux/linux/issues/1210 is fixed.

### CODE REVIEW

File: tuxbuild/mainline.tux.yml [hunk 1]
```diff
@@ -6,13 +6,4 @@
...
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: arm, toolchain: clang-11, kconfig: aspeed_g5_defconfig, make_variables: {LLVM: "1"}}
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: arm, toolchain: clang-11, kconfig: multi_v7_defconfig, make_variables: {LLVM: "1"}}
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: arm64, toolchain: clang-11, kconfig: defconfig, make_variables: {LLVM: "1"}}
-      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: x86_64, toolchain: clang-11, kconfig: defconfig, make_variables: {LLVM: "1"}}
+      - {git_repo: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git", git_ref: "master", target_arch: i386, toolchain: clang-nightly, kconfig: defconfig, make_variables: {LLVM: "1"}}
```
Comment (nathanchance) [inline tuxbuild/mainline.tux.yml:9]: This diff doesn't look right?

Comment (nickdesaulniers) [reply tuxbuild/mainline.tux.yml:9]: oh, yeah, I shouldn't have committed this...will undo, tomorrow

Comment (nickdesaulniers) [reply tuxbuild/mainline.tux.yml:9]: (I was playing around with the file and invoking tuxsuite locally)

Comment (nickdesaulniers) [reply tuxbuild/mainline.tux.yml]: done PTAL

**Comment by nathanchance** [comment]
Does `utils.py` need to be updated to set `x86` for boot-utils?

https://github.com/ClangBuiltLinux/continuous-integration2/blob/2d6610767563026ce56d6763a45f1894b7de6383/utils.py#L58

Or should we just update boot-utils to take i386 instead of x86?

**Comment by nickdesaulniers** [comment]
> Does utils.py need to be updated to set x86 for boot-utils?

Likely, see comment:

>> Still need to add check_logs.py support once
>> ClangBuiltLinux/linux#1210 is fixed.

(I don't expect this to close issue #17; it's just the first half. I'd prefer to fix up utils.py once the build actually produces a binary image remotely that I can test with)

**Review by nathanchance — APPROVED** [review]
Ah fair enough then, this is fine for now.

**Pull request merged**